build/ci/github_actions/driver/wait_for_pods_ready_utils.sh

```diff
@@ -27,7 +27,7 @@ get_image_pod_by_type (){
   done
 }
 
-wait_for_driver_pod_to_start (){
+wait_for_pod_to_start (){
```

Comment:
Using `utils` as part of the filename is too general. I suggest renaming the file

Reply:
`function_to_help_deploy_pods.sh` is better?

Reply:
nop, it has the same generic meaning.

Reply:
`deploy_object.sh`

Reply:
`deployment.sh`?

Reply:
@zingero, what you think is better?
because me and Orian agreed on Slack on `deploy_object.sh`

Reply:
I'm OK with `deploy_object.sh` as well

Reply:
`deployment` is good as well

Reply:
I think it's a bit better since it doesn't sound like an action

Reply:
indeed

Reply:
done